services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py

```diff
@@ -106,7 +106,6 @@ async def docker_compose_create(
     (Re)creates, starts, and attaches to containers for a service
 
     - does NOT build images
```

Comment:
you might completely rewrite that docstring maybe?
it only creates the containers. nothing else.

Reply:
agree with san